### PR TITLE
Fixing syntax errors left over from javascript source copy

### PR DESCRIPTION
**Pull request by aurorawalker**
Attempted to use the Python Readme/getCustomAttributes file when setting up the Battlesnake dashboards, but noticed a few errors that look like they are left over from porting this from the javascript version of this (I assume that one was created first!)

These are the changes I made for myself, I believe these are the correct changes to merge into the mainline for others wanting to use this template for Python.

### CODE REVIEW

File: python/getCustomAttributes.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def getCustomAttributes(data: dict) -> dict:
 
     return attributes
 
-def getCustomAttributes(data: dict) -> dict:
+def getCustomAttributesEnd(data: dict) -> dict:
```
Comment (aurorawalker) [inline python/getCustomAttributes.py:53]: Both functions were mistakenly called `getCustomAttributes()` but of course we want two different functions! This mirrors the name used in the Javascript version of this file, as well as what the README has.

File: python/README.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ To use this helper script, add the [New Relic Python agent](https://docs.newreli
 from flask import Flask
 
 # Add the New Relic APM agent.
-import newrelic
+import newrelic.agent
```
Comment (aurorawalker) [inline python/README.md:11]: The documentation at https://github.com/newrelic/newrelic-python-agent#usage indicates that this is the correct way of importing the agent, and it seemed to work for me locally

File: python/getCustomAttributes.py [hunk 3]
```diff
@@ -65,9 +65,9 @@ def getCustomAttributes(data: dict) -> dict:
         "snakeHealth": you["health"],
         "snakeLength": you["length"],
 
-        "snakeGameWinnerName": board["snakes"][0].name if len(board["snakes"]) > 0 else Null,
-        "snakeGameWinnerId":  board["snakes"][0].id if len(board["snakes"]) > 0 else Null,
-        "snakeGameIsWin": board["snakes"][0].name == you["name"] if len(board["snakes"]) > 0 else False,
+        "snakeGameWinnerName": board["snakes"][0]["name"] if len(board["snakes"]) > 0 else None,
```
Comment (aurorawalker) [inline python/getCustomAttributes.py:68]: Looks like this bit was a Javascript porting error - there is no `Null` in Python, only `None`, also accessing `name` and `id` had to be done via dictionary access via `["name"]`, as they were not object attributes.

Comment (zstix) [reply python/getCustomAttributes.py:68]: 100% a porting error. Writing JS all day has eroded my python skills 😩

**Review by zstix — APPROVED** [review]

**Pull request merged**